app/controllers/slack_controller.rb

```diff
@@ -0,0 +1,27 @@
+require "#{Rails.root}/lib/slack"
+
+class SlackController < ApplicationController
+  include Slack
+
+  def send_notification
+    raise CanCan::AccessDenied unless current_user.admin?
+
+    @user = User.find(params[:id])
+
+    # Get Slack user id by email
+    get_response = SlackRequest.get('users.lookupByEmail', { email: @user.email })
+    slack_user_id = get_response['user']['id']
+    @message_text = "This is a friendly reminder for you to submit your monthly report on VizzTrackr :simple_smile:\nPlease click on the following link to do so: http://vizz-trackr.herokuapp.com/my-report"
+
+    # Post Slack message
+    post_body_data = "{\"channel\": \"#{slack_user_id}\", \"text\": \"#{@message_text}\"}"
+    post_response = SlackRequest.post('chat.postMessage', post_body_data)
+    @notification_success = post_response['ok']
+
+    respond_to do |format|
+      format.json
```

Comment:
If we are always redirecting to the HTML page we don't need the respond_to block here. we can get rid of it, or at least format.json and format.js, as we won't be using those either.

Reply:
Changed this to a redirect, as suggested in the comment below, so this part is no longer needed 👍